can-stache-converters.js

```diff
@@ -61,8 +61,8 @@ stache.registerConverter("index-to-selected", {
 });
 
 stache.registerConverter("either-or", {
-	get: function(chosen, a, b){
-		return b !== chosen();
+	get: function(chosen, a){
+		return a === chosen();
```

Comment:
I think it's important to change this, now. Without this change, the case where `chosen()` doesn't match `a` or `b` results in a checked checkbox without the chosen value being updated to `a` or `b`.